int16 yaw_value
int16 roll_value
int16 pitch_value
int16 duration_value